packages/webapp/src/containers/Animals/Inventory/styles.module.scss

```diff
@@ -29,8 +29,66 @@
   }
 }
 
-.taskViewHeight {
-  height: calc(100vh - var(--global-navbar-height) - var(--global-multi-step-task-layout-aggregated-height));
+.divWrapper {
+  height: auto;
+  overflow-y: auto;
+}
+ 
+.taskViewMaxHeight {
+  max-height: calc(100vh - var(--global-navbar-height) - var(--global-multi-step-task-layout-aggregated-height));
+}
+
+.summaryViewHeight {
+  height: auto;
+}
+
+.section {
+  margin-top: 24px;
+  padding: 8px 0;
+  background: var(--White);
+  border-radius: 8px;
+  border: 1px solid var(--Colors-Neutral-Neutral-50);
+  gap: 8px;
+
+  @include lg-breakpoint {
```

Comment:
Makes sense! Having to use full width routes and apply weird padding styles for some components in some widths got weird.  I will let you do it since you know what you need!